uds/database/service/abstract_service.py

```
@@ -1,14 +1,20 @@
 """Definition of UDS Service data encoding and decoding."""
 
-__all__ = ["AbstractService"]
+__all__ = ["AbstractService", "DataRecordOccurrencesValuesAlias", "DecodedMessageAlias"]
 
 from abc import ABC, abstractmethod
-from typing import List, Union
+from typing import List, Sequence, Union
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)